dbc2val/mapping.md

```diff
@@ -0,0 +1,308 @@
+you must clone the repository, update submodules and then generate VSS JSON like in the example below:
+
+```
+git submodule update --init
+vss-tools/vspec2json.py -e dbc -o dbc_overlay.vspec --no-uuid ./spec/VehicleSignalSpecification.vspec vss_dbc.json
+```
+
+An alternative approach is download a tar archive from an [official VSS release](https://github.com/COVESA/vehicle_signal_specification/releases)
+and use the included Yaml file as base.
+
+```
+vss-tools/vspec2json.py -e dbc -o dbc_overlay.vspec --no-uuid vss_rel_3.0.yaml vss_dbc.json
+```
+
+_Note: The dbc feeder relies on correct VSS information in the JSON file. This means that if KUKSA.val Databroker VSS JSON file updated, then the file used in DBC-feeder possibly needs to be updated as well._
+
+
+## Mapping syntax
+
+The syntax for a DBC definition of a signal in an overlay is specified below.
+The syntax if information instead is added directly to a VSS JSON file is similar, but not described here.
+See [vss_dbc.json](vss_dbc.json) for examples on DBC specification in JSON format.
+Search for `dbc` to find the signals where DBC mapping has been defined.
+If a signal does not have DBC mapping it will be ignored by the DBC feeder.
+
+Syntax
+
+```yaml
+<VSS Signal name>:
+  type: <VSS type>
+  datatype: <VSS datatype>
+  dbc:
+    signal: <DBC signal name>
+    [interval_ms: <interval in milliseconds>]
+    [on_change: {True|False}]
+    [transform: ...]
+```
+
+Specifying DBC signal name is mandatory. It must correspond to a DBC signal name defined in a DBC file.
+By default the DBC feeder use the [Model3CAN.dbc](Model3CAN.dbc) example file.
+
+`interval_ms` and `on_change` are optional and control under which conditions a value shall be forwarded.
+The `interval_ms` value indicates the minimum interval between signals in milliseconds.
+The `on_change: True` argument specifies that the VSS signal only shall be sent if the DBC value has changed.
+If none of them are specified it corresponds to `interval_ms: 1000, on_change: False`.
+If only `on_change: True` is specified it corresponds to `interval_ms: 0, on_change: True`
+
+The `transform` entry can be used to specify how DBC values shall be mapped to VSS values.
+If transform is not specified values will be transformed as is.
+
+### Math Transformation
+
+A Math transformation can be defined by the `math` attribute.
+It accepts [py-expression-eval](https://github.com/AxiaCore/py-expression-eval/) formulas as argument.
+The DBC feeder expects the DBC value to be represented as `x` like in the example below.
+
+```yaml
+Vehicle.OBD.EngineLoad:
+  type: sensor
+  datatype: float
+  dbc:
+    signal: RearPower266
+    interval_ms: 4000
+    transform:
+      math: "floor(abs(x/5))"
+```
+
+Transformation may fail. Typical reasons may include that the DBC value is not numerical,
+or that the transformation fails on certain values like division by zero.
+If transformation fails the signal will be discarded.
```

Comment:
discarded->ignored

Reply:
Edit: I see the `discarded`wording is just often/consistently.  Will not mark every occurrence. can live with it :) I just would have written ignored